CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+
+## [1.51](https://github.com/brave/brave-ios/releases/tag/v1.51)
+
+### Web3
+
+- Added network icons to be shown for native assets. ([#7123](https://github.com/brave/brave-ios/issues/7123))
+- Implemented Market Tab in Brave Wallet. ([#7202](https://github.com/brave/brave-ios/issues/7202))
+- Implemented NFT tab in Brave Wallet. ([#7173](https://github.com/brave/brave-ios/issues/7173))
+- Updated label for "Reset and clear wallet data" under Wallet settings. ([#7337](https://github.com/brave/brave-ios/issues/7337))
+- Fixed Ethereum DApp error response formatting. ([#7392](https://github.com/brave/brave-ios/issues/7392))
+
+### General
+
+- Added VPN region selector in the main menu. ([#7380](https://github.com/brave/brave-ios/issues/7380))
+- Added VPN Protocol options in VPN settings. ([#7255](https://github.com/brave/brave-ios/issues/7255))
+- Implemented built-in iOS 16 find-in-page controls. ([#7238](https://github.com/brave/brave-ios/issues/7238))
```

Comment:
Is the feature only for `iOS 16`? Maybe something like this which is the structure we usually use:

```
- Implemented built-in find-in-page controls on iOS 16 or higher. ([#7238](https://github.com/brave/brave-ios/issues/7238))
```